### PR TITLE
Add house outlay api

### PR DESCRIPTION
**Pull request by ylyangtw**
For testing, run the api locally and you can access the result here: 
http://localhost:5000/pdl/titles/title-ii/programs/house-outlay/predicted
http://localhost:5000/pdl/titles/title-ii/programs/house-outlay/practice-names

### CODE REVIEW

**Review by pengyin-shan — COMMENTED** [review]
@ylyangtw, this looks great! I just have a few questions regarding its usage:

1. Will I need to request the entire JSON from the backend once to retrieve all the data (similar to what we currently have for EQIP-IRA), or should I query using practice codes at the end, like in this example: https://policydesignlab-dev.ncsa.illinois.edu/pdl/titles/title-ii/programs/eqip/state-distribution?practice_codes=300,340,350 for the EQIP selector?

2. Will there be another API endpoint for practice names, similar to https://policydesignlab-dev.ncsa.illinois.edu/pdl/titles/title-ii/programs/eqip-ira/practice-names? This would be necessary to create the selector's menu.

**Review by pengyin-shan — APPROVED** [review]
The new `pratice-name` endpoint looks good to me, approving this PR

File: CHANGELOG.md
Comment (sandeep-ps) [inline CHANGELOG.md:1]: You may need to merge the develop branch to this branch, as release 0.20.0 details are not showing up in the CHANGELOG.

File: README.md
Comment (sandeep-ps) [inline README.md:21]: Good catch!

**Review by sandeep-ps — APPROVED** [review]
Approving with feedback about merging the latest changes from develop to this branch. Other changes look good to me. Thanks. After updating the CHANGELOG, please merge using Squash and Merge. Thanks.

**Pull request merged**
